test/parallel/test-cluster-worker-disconnect.js

```diff
@@ -86,7 +90,7 @@ if (cluster.isWorker) {
 
     // flags
     assert.equal(w.state, 'disconnected', 'The state property was not set');
-    assert.equal(w.suicideMode, true, 'Suicide mode was not set');
+    assert.equal(w.voluntaryMode, true, 'Voluntary exit mode was not set');
```

Comment:
This file still has `voluntary exit` and `voluntary` references. It's only a test file, though.